node-src/lib/e2e.ts

```diff
@@ -59,3 +59,7 @@ export async function getE2eBuildCommand(
     throw err;
   }
 }
+
+export function isE2EBuild(options: Options) {
+  return options.playwright || options.cypress;
+}
```

Comment:
Nice. We could use this for the "flag the project as e2e" thing in the future (if we ever feel the need to handle projects switching from storybook-e2e or vice versa).